### PR TITLE
Bump all formula versions using protobuf

### PR DESCRIPTION
**Pull request by osrfbuild**
A protobuf update broke all the bottles using protobuf as a dependency. Bump all revisions to trigger new bottle builds.

### CODE REVIEW

File: haptix-comm.rb [hunk 1]
```diff
@@ -5,6 +5,8 @@ class HaptixComm < Formula
   sha256 "d495f65e401fc9e3c8fcbdded347313287c2de55f1605f6b76d02a26b893c08d"
   head "https://bitbucket.org/osrf/haptix-comm", :branch => "default", :using => :hg
 
+  revision 1
```
Comment (scpeters) [inline haptix-comm.rb]: we don't have bottles for this, so it doesn't need a bump

**Comment by j-rivero** [comment]
Looks like Travis said enough 
`The job exceeded the maximum time limit for jobs, and has been terminated.`

@scpeters do you think it is ok to merge?

**Comment by scpeters** [comment]
el_capitan bottles?

**Comment by j-rivero** [comment]
> el_capitan bottles?

I know I was forgetting something. I go for it.

**Comment by scpeters** [comment]
looks good to me, thanks!

**Pull request merged**